File: SpringPart/src/main/java/springPart/model/ispit/PrijavaIspita.java
package springPart.model.ispit;

import java.time.LocalDateTime;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

import springPart.model.korisnik.Student;

@Entity
public class PrijavaIspita {
	@Id
	Long id;
	
	LocalDateTime datumPrijave;
	
	@ManyToOne
	Student student;
	
	@ManyToOne
	Rok rok;
	
	@ManyToOne
	Ispit ispit;
	
	
}

File: AngularPart/src/app/Models/prijava-ispita.ts
import { Student } from './student';
import { Rok } from './rok';
import { Ispit } from './ispit';

export interface PrijavaIspita {
    id:number;
    datumPrijave:Date;
    student:Student;
    rok:Rok;
    ispit:Ispit
}
